bcgov/GDX-Analytics
# upgrade braces to v3.0.3

This PR updates the package-lock.json with a fresh npm install and bumps braces to version: 3.0.3.
Merging this PR should fix the following:
https://github.com/bcgov/GDX-Analytics/security/dependabot/150

Testing instructions:
- clone the repo and switch to dependant branch
```
git clone https://github.com/bcgov/GDX-Analytics.git
cd GDX-Analytics/
git checkout GDXDSD-7054-react-tracker-branches
```
- install react and related packages globally
```
npm install -g react react-dom react-router-dom
```
- install the dependencies listed in the package-lock.json
```
cd examples/react_tracker/
npm ci
```
- start a development server
```
npm start
```
check if sample react.js page is running by browsing to http://localhost:3000/
